tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -1,122 +1,170 @@
 ---
 title: Console Audit Logging
-summary: Learn about the log auditing feature for the TiDB Cloud console.
+summary: Learn about the audit logging feature for the TiDB Cloud console.
 ---
 
 # Console Audit Logging
 
-TiDB Cloud provides the audit logging feature for TiDB Cloud console operations, which records the history of user access details (such as user login to the Console and cluster creation operations).
+TiDB Cloud provides the console audit logging feature to help you track various behaviors and operations of users on the [TiDB Cloud console](https://tidbcloud.com). For example, you can track operations, such as inviting a user to join your organization and creating a cluster.
 
-> **Note:**
->
-> Currently, the **audit logging** feature is experimental. The output is subject to change.
+## Prerequisites
 
-## Limitations
+- You must be in the Owner or Audit Admin role of your organization in TiDB Cloud. Otherwise, you cannot see the console audit logging-related options in the TiDB Cloud console. The Audit Admin role is only visible upon request, so it is recommended that you use the Owner role directly. If you need to use the Audit Admin role, click **Help** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com), fill in "Apply for the Audit Admin role" in the **Description** field, and then click **Send**. For more information about roles in TiDB Cloud, see [Manage role access](/tidb-cloud/manage-user-access.md#manage-role-access).
+- You can only enable and disable the console audit logging for your organization. You can only track the actions of users in your organization.
+- After the console audit logging is enabled, all event types of the TiDB Cloud console will be audited, and you cannot specify only auditing some of them.
 
-Currently, the console audit logging feature has the following limitations:
+## Enable console audit logging
 
-- Console audit logging is enabled by default and cannot be disabled by users.
-- You cannot specify the audit filtering rules for the feature.
-- You can access the audit logs only with the help of PingCAP support.
+The console audit logging feature is disabled by default. To enable it, take the following steps:
 
-## Audit event types
+1. In the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> **Organization** > **Console Audit Logging**.
+2. Click **Enable Console Audit Logging**. Or click **Setting** in the upper-right corner to enable console audit logging.
```

Comment:
```suggestion
2. If it is the first time your organization enables console audit logging, click **Enable Console Audit Logging**. Otherwise, click **Setting** in the upper-right corner to enable console audit logging.
```